src/components/list/examples/list-grid.tsx

```diff
@@ -0,0 +1,61 @@
+import { ListItem } from '@limetech/lime-elements';
+import { Component, h } from '@stencil/core';
+
+/**
+ * List with grid layout
+ * To display list items in a grid layout instead of a vertical column,
+ * simply add `has-grid-layout` class to
+ * the component.
+ *
+ * This layout can be customized, using a few CSS variables.
```

Comment:
How can the layout be customised? 😛 The CSS variables are described in the menu component, but shouldn't they be described in the list component (here) instead? Maybe describe them here and then mention that they are described here in the menu component? 🤔

Reply:
> How can the layout be customised?

What should I say instead? :) I think when you tweak the distances or max/min-width, it's "customizing" the layout. Maybe better to say "tweaking" or "adjusting" the layout? 

> but shouldn't they be described in the list component (here) instead?

You're right. But they are already explained _both_ in list and _menu_, because they can be tweaked in both components. But menu only lets users to customize some, coming from the list. For example, `--list-grid-max-width` is only available in list, because it doesn't make sense in menu.

List: 
![image](https://user-images.githubusercontent.com/35954987/128867025-cb41aa82-5222-44a8-88d0-715973252a63.png)

Menu:
![image](https://user-images.githubusercontent.com/35954987/128867090-5a326112-bfdf-4ac0-aeee-bbbc5a69e1d3.png)

Reply:
I kind of forgot that the styles are explained in the styles section 😅 So then I think it looks good! 🙂